OmniBLE/PumpManager/OmniBLEPumpManager.swift

```diff
@@ -1845,7 +1840,7 @@ extension OmniBLEPumpManager: PumpManager {
 
         switch shouldFetchStatus {
         case .none:
-            completion?(lastSync)
+            completion?(self.lastSync)
```

Comment:
nit: not a fan of using self where it's not needed, but not a blocker, for sure; just a style thing.

Reply:
Just 5 and 9 lines down (on lines 1848 and 1852) there are two others cases within this switch statement and they both use the self.lastSync style. I just thought it was better to have them all match within a single switch statement and I just preferred the more explicit self style used in the other two cases here.